### PR TITLE
[TR] PIM-5375: mv /upgrades from std to dev

### PR DESCRIPTION
**Pull request by willy-ahva**
In order to not have anymore to move and adapt migration scripts from dev to std for each releases, we decided to create a pre-update-cmd script for composer which will copy `upgrades/` folder from dev to std.
So we have to repatriate all the `upgrades/` folder in dev.
Then, do the composer script in the dev version.
Linked to : https://github.com/akeneo/pim-community-dev/pull/3955

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -57,7 +57,8 @@
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::removeSymfonyStandardFiles",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget",
-            "php app/console fos:js-routing:dump --target=web/js/routes.js"
+            "php app/console fos:js-routing:dump --target=web/js/routes.js",
+            "Pim\\Bundle\\InstallerBundle\\ComposerScripts::copyUpgradesDirFromDev"
```
Comment (solivier) [inline composer.json]: this will be called after each composer update ?

Comment (willy-ahva) [inline composer.json]: yes

File: app/AppKernel.php [hunk 2]
```diff
@@ -130,7 +130,7 @@ protected function getSymfonyBundles()
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
             new Doctrine\Bundle\DoctrineBundle\DoctrineBundle(),
             // Uncomment the following line to use MongoDB implementation
-            // new Doctrine\Bundle\MongoDBBundle\DoctrineMongoDBBundle(),
+             new Doctrine\Bundle\MongoDBBundle\DoctrineMongoDBBundle(),
```
Comment (jjanvier) [inline app/AppKernel.php]: to revert

File: app/config/pim_parameters.yml [hunk 3]
```diff
@@ -29,11 +29,11 @@ parameters:
     max_products_category_removal: 100
 
     # Product storage driver can be doctrine/orm or doctrine/mongodb-odm
-    pim_catalog_product_storage_driver: doctrine/orm
```
Comment (jjanvier) [inline app/config/pim_parameters.yml]: all to revert

File: composer.json [hunk 4]
```diff
@@ -19,7 +19,7 @@
         }
     },
     "require": {
-        "akeneo/pim-community-dev": "v1.5.0-ALPHA1@alpha",
+        "akeneo/pim-community-dev": "dev-PIM-5375",
```
Comment (willy-ahva) [inline composer.json]: revert too

File: app/config/pim_parameters.yml [hunk 5]
```diff
@@ -33,7 +33,7 @@ parameters:
 
     ## Uncomment to configure mongodb
     # mongodb_server: 'mongodb://localhost:27017'
-    # mongodb_database: akeneo_pim
+    # mongodb_database: ce_standard_mongo
```
Comment (jjanvier) [inline app/config/pim_parameters.yml]: to revert

**Comment by jjanvier** [comment]
gtm once fixed
squash your commits please

**Comment by nidup** [comment]
gtm

**Pull request merged**